### PR TITLE
drop chef from the nightly manual

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: manuals/nightly/index.md [hunk 1]
```diff
@@ -192,27 +192,24 @@ previous_version: "3.0"
...
-## 4.10 Monitoring
-{%include manuals/{{ page.version }}/4.10_monitoring.md %}
-### 4.10.1 Global status
-{%include manuals/{{ page.version }}/4.10.1_global_status.md %}
-### 4.10.2 Sub-statuses
-{%include manuals/{{ page.version }}/4.10.2_substatuses.md %}
-### 4.10.3 Searching by statuses
-{%include manuals/{{ page.version }}/4.10.3_searching_by_statuses.md %}
-
-## 4.11 Reports
```
Comment (ekohl) [inline manuals/nightly/index.md]: You'll hate me for this, but this breaks anchors in Foreman:
https://github.com/theforeman/foreman/blob/d559505b68f5461593ecb945f602a7925f260179/app/controllers/report_templates_controller.rb#L10
https://github.com/theforeman/foreman/blob/d559505b68f5461593ecb945f602a7925f260179/app/views/report_templates/welcome.html.erb#L9
https://github.com/theforeman/foreman/blob/d559505b68f5461593ecb945f602a7925f260179/app/views/report_templates/index.html.erb#L4

Comment (evgeni) [reply manuals/nightly/index.md]: I have no idea how to answer that :smiling_imp:

Comment (ekohl) [reply manuals/nightly/index.md]: It means we have to patch Foreman if we reorder this (change 411 to 410).

Comment (evgeni) [reply manuals/nightly/index.md]: Or we just replace the section with "the chef plugin was removed"? Then there is no reordering

Comment (ekohl) [reply manuals/nightly/index.md]: I'd be OK with that in the short term. Long term we'd ideally migrate to the new docs.theforeman.org anyway.

Comment (evgeni) [reply manuals/nightly/index.md]: done now

**Review by ekohl — APPROVED** [review]

**Pull request merged**